### DISCUSSION TITLE
Issue in building tmezone

### DISCUSSION THREAD

**Discussion by NitinMehra0019** [valhalla/valhalla#4597] [Q&A]
Hi there, 
I found a issue when I am trying to build time zone during the configuration of Valhalla as mentioned in https://gis-ops.com/valhalla-part-2-how-to-run-valhalla-on-ubuntu/

I am attaching the issue 

```bash
jangoo@jangoo-pc2:~/valhalla$ ./scripts/valhalla_build_timezones > ./valhalla_tiles/timezones.sqlite
bash: ./valhalla_tiles/timezones.sqlite: No such file or directory
```
Its showing No such file or directory even though there is file exist with name timezones.sqlite.

**Comment by nilsnolde** [comment]
Dude, come on haha of course you’ll need to create the directory first. Sorry but this is all too obvious..

**Reply by kevinkreiser** [reply]
this is the comedy gold :pray:

**Comment by michaelkirk** [comment]
Looks like you forgot the `mkdir valhalla_tiles && mkdir conf` step from the tutorial you linked.
